diesel/src/pg/types/ranges.rs

```diff
@@ -24,41 +24,51 @@ bitflags::bitflags! {
     }
 }
 
-#[cfg(feature = "postgres_backend")]
-impl<ST: 'static, T> AsExpression<Range<ST>> for (Bound<T>, Bound<T>) {
-    type Expression = SqlBound<Range<ST>, Self>;
-
-    fn as_expression(self) -> Self::Expression {
-        SqlBound::new(self)
-    }
+macro_rules! range_as_expression {
+    ($ty:ty; $sql_type:ty) => {
+        #[cfg(feature = "postgres_backend")]
+        // this simplifies the macro implementation
+        // as some macro calls use this lifetime
+        #[allow(clippy::extra_unused_lifetimes)]
+        impl<'a, 'b, ST: 'static, T> AsExpression<$sql_type> for $ty {
```

Comment:
Can you point out where `'b` is used?

Reply:
Well notice, I forgot that there. Thanks!